### PR TITLE
Release/1.0.3

### PR DESCRIPTION
**Pull request by usame-algan**
## Includes

- #1038 
- #1032 
- #1041

### CODE REVIEW

**Review by iamacook — APPROVED** [review]

File: src/services/analytics/useGtm.ts [hunk 1]
```diff
@@ -19,7 +19,10 @@ const useGtm = () => {
 
   // Initialize GTM, or clear it if analytics is disabled
   useEffect(() => {
-    isAnalyticsEnabled ? gtmInit() : gtmClear()
+    // router.pathname doesn't contain the safe address
+    // so we can override the initial dataLayer
+    isAnalyticsEnabled ? gtmInit(router.pathname) : gtmClear()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (katspaugh) [inline src/services/analytics/useGtm.ts:25]: Not the best solution, we should use a state variable here that would be assigned to the initial pathname only once.

**Review by katspaugh — APPROVED** [review]
🔥

**Comment by francovenica** [comment]
The link in the footer is fine. The 1.0.3 page in github doesn't exist yet, but when it does it will be fine
The page view is fine

Regarding the Address in the tracker, there is the referral still showing the full address.
I understand that referrals could be any page in the world, but since is being tracked as we move through the site we should parse it and check there is no address present
image

**Comment by usame-algan** [comment]
> Regarding the Address in the tracker, there is the referral still showing the full address.

Afaik we had the referrer overwrite in GTM at some point but removed it again. Testing on localhost, the referrer doesn't seem to be there in the events but on prod it is. I suggest we merge this as it anonymizes the path and location and look at the referrer issue separately.

**Pull request merged**